ros2topic/ros2topic/verb/echo.py

```diff
@@ -218,9 +221,14 @@ def _subscriber_callback(self, msg, info):
             for fields in self.fields_list:
                 submsg = msg
                 for field in fields:
+                    is_indexing = re.compile(r'^\[(\d+)\]$')
```

Comment:
```suggestion
                   # Matches strings exactly in the format "[digits]" (e.g., "[123]")
                    is_indexing = re.compile(r'^\[(\d+)\]$')
```
Make life easy for the next person. ;)

Reply:
Added the comment also a few lines above